src/sentry/snuba/metrics/utils.py

```diff
@@ -128,7 +130,8 @@
         "count": "countIf",
         "max": "maxIf",
         "min": "minIf",
-        "p50": "quantilesIf(0.50)",  # TODO: Would be nice to use `quantile(0.50)` (singular) here, but snuba responds with an error
+        "p50": "quantilesIf(0.50)",
+        # TODO: Would be nice to use `quantile(0.50)` (singular) here, but snuba responds with an error
```

Comment:
Maybe you mis-typed this.

Reply:
Haven't touched it.... it appears as changed due to an automatic re-format.